src/connectors/AuthServerConnector.ts

```diff
@@ -11,16 +11,16 @@ export class AuthServerConnector extends AbstractConnector {
   }
 
   activate = async (): Promise<ConnectorUpdate<string | number>> => {
-    const identity = this.provider.getIdentity()
+    const account = this.provider.getAccount()
```

Comment:
To make the getIdentity function private, I can use the getAccount result to determine if the user has signed in already. Both the identity and the account are stored and deleted on the same flows, so the existence of one determines the existence of the other. (Unless one is manually deleted from storage)